test/other/codesize/test_codesize_minimal_pthreads.funcs

```diff
@@ -72,6 +72,7 @@ $emscripten_futex_wake
 $emscripten_stack_get_current
 $emscripten_stack_set_limits
 $free_ctx
+$get_or_add_tasks_for_thread
```

Comment:
I wonder what happened here.

Reply:
I guess this was previously inlined in do_proxy?